lib/itamae/backend.rb

```diff
@@ -292,7 +292,9 @@ def finalize
         /\A(?<repo>.+?)(?:|:(?<tag>[^:]+))\z/.match(@options[:tag]) do |m|
           image.tag(repo: m[:repo], tag: m[:tag])
         end
-        Itamae.logger.info "Image created: #{image.id}"
+        log_message = "Image created: #{image.id}"
+        log_message << ", and tagged as #{@options[:tag]}"
```

Comment:
I got this log message in not passed `--tag` option.

```
INFO : Image created: sha256:5647ea2e1d250a3fd526ee434c4ae3bcbaabd9797bbbcba9077bab39a759e0a7, and tagged as
```

I think more better to add guard clause to log message build phase. :wink: 

```suggestion
        log_message << ", and tagged as #{@options[:tag]}" if @options[:tag]
```

Reply:
Thank you! :+1: :+1: It makes sence absolutely.
I was going to write the conditon, but it looks I forgot the condition. :innocent: